code/__DEFINES/xeno.dm

```diff
@@ -177,6 +177,17 @@
 /// The time until you can re-corrupt a comms relay after the last pylon was destroyed
 #define XENO_PYLON_DESTRUCTION_DELAY (5 MINUTES)
 
+/// Evolution boost during hijack
+#define XENO_HIJACK_EVILUTION_BUFF 10
+
+/// For how long the buff lasts
+#define XENO_HIJACK_EVILUTION_TIME 3 MINUTES
```

Comment:
use ( ) around time defines to avoid people tripping up on math operators - like XENO_PYLON_DESTRUCTION_DELAY above